source/opt/function.h

```diff
@@ -88,6 +88,10 @@ class Function {
   // Returns the entry basic block for this function.
   const std::unique_ptr<BasicBlock>& entry() const { return blocks_.front(); }
 
+  // Returns the last basic block in this function.
+  BasicBlock* back() { return blocks_.back().get(); }
+  const BasicBlock* back() const { return blocks_.back().get(); }
```

Comment:
`BasicBlock` uses `tail()` instead of `back()`.
For the consistency of the name, why don't we use `tail()`?

Reply:
Done.  Good call.